### PR TITLE
KEYCLOAK-7094 Support redirect to external logout page

### PR DESCRIPTION
**Pull request by hmlnarik**
Related to keycloak/keycloak#5234

### CODE REVIEW

File: securing_apps/topics/saml/java/general-config/sp_element.adoc [hunk 1]
```diff
@@ -56,3 +56,9 @@ autodetectBearerOnly::
   but send an HTTP `401` status code to unauthenticated SOAP or REST clients instead as they would not understand a redirect to the login page.
   Keycloak auto-detects SOAP or REST clients based on typical headers like `X-Requested-With`, `SOAPAction` or `Accept`.
   The default value is _false_.
+
+logoutPage::
+  This sets the page to display after logout. If the page is full URL (i.e. it includes scheme part, e.g. `http://web.example.com/logout.html`),
+  the user is redirected after logout to that page using HTTP `302` status code. In case a plain page name is specified (e.g. `/logout.jsp`), the page
+  is displayed after logout _regardless of whether it lies in protected area according to `security-constraint` declarations in web.xml_. The page is
+  resolved relative to the deployment context root.
```
Comment (matthewhelmke) [inline securing_apps/topics/saml/java/general-config/sp_element.adoc]: logoutPage::
  This sets the page to display after logout. If the page is a full URL, such as `http://web.example.com/logout.html`,  the user is redirected after logout to that page using the HTTP `302` status code. If a relative link is specified, such as `/logout.jsp`, the page  is displayed after logout, _regardless of whether it lies in a protected area according to `security-constraint` declarations in web.xml_, and the page is resolved relative to the deployment context root.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
I have an edited version ready for you with some minor changes.

File: securing_apps/topics/saml/java/general-config/sp_element.adoc (hunk 1 above)
Comment (hmlnarik) [reply securing_apps/topics/saml/java/general-config/sp_element.adoc]: Thanks @matthewhelmke!

**Review by patriot1burke — COMMENTED** [review]
Looks good.  Awaiting build passing then will approve

**Review by matthewhelmke — APPROVED** [review]
LGTM, thank you.

**Pull request merged**